docs/updates/2019-09.md

```diff
@@ -0,0 +1,110 @@
+# September 2019
+
+The summer has passed in the northern hemisphere and it's time for a new Qlik Core update. There are a number of
+updates in this version that we hope you will like, some of the key highlights include:
+
+## New license key format
+
+We're migrating to a new license format effectively dropping the serial and control number in favor of a license-key.
+For you running on trial license, you will get the new key when you apply for a renewal and existing customer's will
+receive a new key.
+
+Please read more about how to apply the license key in the
+[core-using-license](https://github.com/qlik-oss/core-using-licenses) repo.
+
+## Release of engima-go 1.0
+
+Similar to [enigma.js](https://github.com/qlik-oss/enigma.js), [enigma-go](https://github.com/qlik-oss/enigma-go) is a
+library that helps you communicate with a Qlik Associative Engine when creating [golang](https://golang.org/) projects.
+Examples of use may be building your own analytics tools, back-end services, or other tools communicating with a Qlik
+Associative Engine.
+
+The library has been alive for quite some time by know and is being used in a couple of external tools.
+Since we believe the quality is there, we decided to fix the API and move the library into a stable
+[1.0 release](https://github.com/qlik-oss/enigma-go/releases/tag/v1.0.0).
+
+If you are interested in how it can be used, take a look at our [corectl](https://github.com/qlik-oss/corectl) tool
+written in golang which uses this as communication layer to the Qlik Associative Engine.
```

Comment:
Maybe also refer to the `enigma-go` examples? https://github.com/qlik-oss/enigma-go/tree/master/examples